keps/sig-node/4815-dra-partitionable-devices/README.md

```diff
@@ -92,9 +101,14 @@ allocated, rather than requiring them to be created before.
 
 ## Motivation
 
-One of the primary motivating examples for supporting partitionable devices
-with DRA is to enable the dynamic allocation of Multi-Instance GPUs
-(MIG) on NVIDIA hardware. MIG devices are represented as fixed-size partitions
+We have several motivating examples for supporting partitionable devices
+with DRA, with the first two described in detail in this document:
+* Partitioning a single GPU into smaller partitions.
+* Multi-host scheduing of interconnected TPUs.
+* SR-IOV
```

Comment:
Add:
* Multi-host logical groupings to avoid deadlocks in LeaderWorkSet scheduling
* Single-host allocations of multiple TPUs in valid topologies
* Single-host allocations of multiple Inferentia and Trainium device in valid topologies